demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -762,7 +762,7 @@ def handle_incorrect_reference_value(self, task_id, values, task_name, section_n
         Returns: True if the references are correct
         """
         is_valid = True
```

Comment:
Can this method (`handle_incorrect_reference_value`) be covered with a unit test?